data_ingest/regs/va.py

```diff
@@ -32,41 +41,53 @@ def load_va_regulations(sleep_time=1):
     sleep_time : int
         The amount of time to sleep between calls to the registry website
     """
-    _connect()
```

Comment:
I think the docstring of this function needs to be updated because we're not using Postgres anymore right?

Reply:
Good catch, thanks!